examples/speech_to_speech/asr_bleu/asr_model_cfgs.json

```diff
@@ -0,0 +1,198 @@
+{
+    "en": {
+        "oct22": {
+            "desc": "Wav2Vec 2.0 Large (LV-60) + Self Training from https://github.com/facebookresearch/fairseq/tree/main/examples/wav2vec#pre-trained-models",
+            "ckpt_path": "https://dl.fbaipublicfiles.com/fairseq/wav2vec/wav2vec_vox_960h_pl.pt",
+            "dict_path": "https://dl.fbaipublicfiles.com/fairseq/wav2vec/dict.ltr.txt",
+            "model_type": "fairseq",
+            "lang": "en",
+            "post_process": "collapse"
+        }
+    },
+    "hok": {
+        "oct22": {
+            "desc": "Hokkien ASR model, for details check [TODO add paper link]",
```

Comment:
Is there a link? Maybe this [one](https://research.facebook.com/publications/hokkien-direct-speech-to-speech-translation/)?

Reply:
Thank you so much @uralik ! LGTM!

Reply:
ah yes, now link exists, will add that!